src/backend/base/langflow/components/jigsawstack/vocr.py

```
@@ -0,0 +1,107 @@
+from typing import Any
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)